components/ProductInfo/ProdImage.js

```diff
@@ -48,54 +52,62 @@ const ImgBtn = ({ image }) => {
 };
 
 const ShareBtn = () => {
-  const [position, setPosition] = useState({ display: "none" });
-
-  let trackMp = [];
-  const displayTooltip = (e) => {
-    trackMp.push([e.pageX, e.pageY]);
-    const left = e.pageX + 10;
-    const top = e.pageY + 12;
-    console.log(trackMp);
-
-    setTimeout(() => {
-      if (
-        trackMp.length >= 2 &&
-        trackMp[trackMp.length - 2][0] === left - 10 &&
-        trackMp[trackMp.length - 2][1] == top - 12
-      ) {
-        setPosition({ display: "block", left, top });
-        setTimeout(() => {}, 1000);
-      }
-    }, 1000);
-
-    if (window.pageX === left - 10 && window.pageY === top) {
-      setPosition({ display: "block", left, top });
-    }
+  const [copyText, setCopyText] = useState("Copy Link");
+  const [linksDisplay, setLinksDisplay] = useState({ display: "none" });
+  const copyUrl = useSelector(({ prodInfo }) => prodInfo.pageUrl);
+  const copyLink = () => {
+    navigator.clipboard
+      .writeText(copyUrl)
+      .then(() => setCopyText("Link Copied!"))
+      .then(() => setTimeout(() => setCopyText("Copy Link"), 1000));
```

Comment:
Created function to copy url to users clipboard on a click in the in the Copy Link a tag